using.regex.py

```diff
@@ -0,0 +1,16 @@
+import re

```

Comment:
Could you please use this function template in the task?

```
def my_function(text, multiplier=2):
    # implement me
    return text
```

Reply:
Done